src/frontend/web_application/src/scenes/ContactBook/components/ContactList/index.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { v1 as uuidV1 } from 'uuid';
-import BlockList from '../../../../components/BlockList';
 import Title from '../../../../components/Title';
 import ContactItem from './components/ContactItem';
+
 import './style.scss';
 
+import { SORT_VIEW_FAMILY_NAME, SORT_VIEW_GIVEN_NAME, SORT_VIEW_TITLE, DEFAULT_SORT_VIEW } from '../../../ContactBook/presenter';
```

Comment:
ok may be this proptype can be omitted since it is a reference to child component ... but well, old plain string will do the job